packages/vue/src/components/organisms/SfHeader/SfHeader.md

```diff
@@ -21,23 +26,25 @@ Header component for page navigation.
   <SfHeader
     title="Storefront UI"
     logo="/assets/logo.svg"
-    active-sidebar="activeSidebar"
+    active-icon="account"
   >
     <template #navigation>
-      <SfHeaderNavigationItem>WOMAN</SfHeaderNavigationItem>
-      <SfHeaderNavigationItem>MAN</SfHeaderNavigationItem>
+      <div class="sf-header__navigation-item">
```

Comment:
Why you removed internal component?

Reply:
back here: https://github.com/DivanteLtd/storefront-ui/pull/472/commits/821c9ae64f1567525efecfd09b85e17acd6a48e8